superset/databases/schemas.py

```diff
@@ -413,11 +413,15 @@ class DatabaseRelatedObjectsResponse(Schema):
     dashboards = fields.Nested(DatabaseRelatedDashboards)
 
 
+class DatabaseFunctionNamesResponse(Schema):
+    function_names = fields.List(fields.String())
+
+
 class ImportV1DatabaseExtraSchema(Schema):
     metadata_params = fields.Dict(keys=fields.Str(), values=fields.Raw())
     engine_params = fields.Dict(keys=fields.Str(), values=fields.Raw())
     metadata_cache_timeout = fields.Dict(keys=fields.Str(), values=fields.Integer())
-    schemas_allowed_for_csv_upload = fields.List(fields.String)
+    schemas_allowed_for_csv_upload = fields.List(fields.String())
```

Comment:
should this change be here? Also how is `fields.Str()` different from `fields.String()`?

Reply:
It should be `String()` here. `Str` is an alias of `String` (see bottom of https://marshmallow.readthedocs.io/en/stable/_modules/marshmallow/fields.html).